packages/sitecore-jss-nextjs-editing-host/src/editing-middleware.ts

```diff
@@ -8,7 +8,7 @@ import { HtmlProcessor } from './html-processors';
 export class EditingMiddleware {
   /**
```

Comment:
Seems like this JSDoc comment should actually be moved from the constructor to the class?

Reply:
@ambrauer Moved description to class, but arguments above constructor

Reply:
@ambrauer https://jsdoc.app/howto-es2015-classes.html